tensor2tensor/data_generators/style_transfer.py

```diff
@@ -69,7 +69,10 @@ def source(self):
     raise NotImplementedError()
 
   def dataset_url(self, dataset_split):
-    raise NotImplementedError()
+    train = dataset_split == problem.DatasetSplit.TRAIN
```

Comment:
Thanks for the PR @tlatkowski !

I can see why you pulled out these 4 lines into the base class, but the base class, as it is written now is should be agnostic about the actual dataset -- no? i.e. I liked your earlier style better, can we revert to that? It only costs us 4 lines.

Another solution would be to create yet another class, maybe `StyleTransferProblemShakespeare` (which is a base class of `StyleTransferProblem`) and have the `StyleTransferShakespeareToModern` and `StyleTransferModernToShakespeare` inherit from it -- however this is unnecessary complexity I feel.

WDYT?

Reply:
Another thing struck me, just renaming `StyleTransferProblem` to `StyleTransferProblemShakespeare` -- will be good too, since the `generate_samples` maybe very specific as well.

LMK. Thanks!